.travis.yml

```diff
@@ -2,7 +2,7 @@ language: scala
 
 scala:
 - 2.12.10
-- 2.13.0
```

Comment:
I remember there was an issue with mima and 2.13.1. Maybe it is fixed in new mima version

Reply:
hmm, that's why it wasn't changed. It seems it worked though